Platform/ApollolakeBoardPkg/Library/Stage1BBoardInitLib/Stage1BBoardInitLib.inf

```diff
@@ -21,6 +21,9 @@
 
 [Sources]
   Stage1BBoardInitLib.c
+  SideBandLib.c

```

Comment:
I think we should keep them as separate libarary in Silicon/ApolloLakeSiliconPkg/Library/IpcLib